pkg/build/api/validation/validation_test.go

```diff
@@ -670,13 +670,14 @@ func TestValidateSource(t *testing.T) {
 		// 3
 		{
 			t:    fielderrors.ValidationErrorTypeInvalid,
-			path: "binary",
+			path: "git",
 			source: &buildapi.BuildSource{
 				Git: &buildapi.GitBuildSource{
 					URI: "https://example.com/repo.git",
 				},
 				Binary: &buildapi.BinaryBuildSource{},
 			},
+			multiple: true,
```

Comment:
i have to assume some new validation tests are in order using image-sourced buildconfigs, even if  they are only valid ones.